Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -46,10 +46,37 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
         return data.trackerAllowlist.state == PrivacyConfigurationData.State.enabled ? data.trackerAllowlist.entries : [:]
     }
     
-    public func isEnabled(featureKey: PrivacyFeature) -> Bool {
+    func parse(versionString: String) -> [Int] {
+        return versionString.split(separator: ".").map { Int($0) ?? 0 }
+    }
+    
+    func staisfiesMinVersion(feature: PrivacyConfigurationData.PrivacyFeature,
+                             versionProvider: AppVersionProvider) -> Bool {
+        if let minSupportedVersion = feature.minSupportedVersion,
+           let appVersion = versionProvider.appVersion() {
+            let minVersion = parse(versionString: minSupportedVersion)
+            let currentVersion = parse(versionString: appVersion)
+            if minVersion.count != currentVersion.count {
+                // Config error
+                return false
```

Comment:
If we keep this check, we'll have to make sure we don't swap to a four-component version number on the macOS browser, like we use for iOS. For safety, it might be worth handling 3 and 4 component version numbers, and just treat the fourth component of a 3 component number as `0`?